Dockerfile

```diff
@@ -10,6 +10,8 @@ COPY . ./
 
 USER root
 
+RUN chmod g+w yarn.lock 
```

Comment:
Not 100% sure why, but I needed this to get around permissions with the lockfile.

Reply:
We where getting an error before so we removed the yarn.lock file.  This looks like this is a better solution.  If this works then let's go ahead and add the yarn.lock file back.   I'm not sure why it's needed either.